backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -86,12 +86,14 @@ private void init() {
   @PostMapping("/link/verify")
   public PxpVerifyResponse getPatientLinkVerify(
       @RequestBody PxpRequestWrapper<Void> body, HttpServletRequest request) {
-    UUID plid = UUID.fromString(body.getPatientLinkId());
-    PatientLink pl = _pls.getPatientLink(plid);
-    OrderStatus os = pl.getTestOrder().getOrderStatus();
-    Person p = _pls.getPatientFromLink(plid);
+    PatientLink pl = _contextHolder.getPatientLink();
+    OrderStatus os = _contextHolder.getLinkedOrder().getOrderStatus();
+    Person p = _contextHolder.getPatient();
     PatientPreferences pp = _ps.getPatientPreferences(p);
-    TestEvent te = _tes.getLastTestResultsForPatient(p);
+    TestEvent te =
+        os == OrderStatus.COMPLETED
+            ? _contextHolder.getLinkedOrder().getTestEvent()
+            : _tes.getLastTestResultsForPatient(p);
```

Comment:
This is a pretty weird behavior, honestly. Is this something we're sure we did intentionally, or do we just have a regression test because it's the current behavior and that's what regression tests protect?

Reply:
Yeah. So, `lastTest` was always part of the PxpVerifyResponse because it's used in the AoE form to ask "was this your last test?" And we just naively _reused_ it to display test results since we already had it in the front end. 

I'm adding a follow-up issue to clean this up, bc they should not piggyback in the same response attribute.